composer.json

```diff
@@ -11,8 +11,14 @@
             "email": "andrei@baibaratsky.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
```

Comment:
I think that wasn’t because of the rollbar package. Please remove these lines and I’ll merge the PR.